fetch.bs

```diff
@@ -5343,6 +5377,13 @@ method, must run these steps:
  {{Headers}} object whose <a for=Headers>guard</a> is
  "<code>immutable</code>".
 
+ <li><p>Add the following steps to <var>requestObject</var>'s <a for=Request>signal</a>'s
+ <a>cancelation callbacks</a>:
```

Comment:
Something like that is how I'd expect it to work. "cancelation callbacks" should probably have for=signal or some such, but I suspect we'll end up renaming these terms a bit before it's all done.